apps/partner2/test/index.coffee

```diff
@@ -7,12 +7,14 @@ Backbone = require 'backbone'
 CurrentUser = require '../../../models/current_user.coffee'
 Profile = require '../../../models/profile.coffee'
 
-newLayoutPartnerProfiles = _.map ['gallery_one', 'gallery_two', 'gallery_three'], (layout) ->
+newLayoutPartnerProfiles = _.map ['gallery_one', 'gallery_two', 'gallery_three' ], (layout) ->
   new Profile fabricate 'partner_profile', owner: fabricate 'partner', profile_layout: layout
 
-deprecatedLayoutPartnerProfiles = _.map ['institution', 'gallery_default', 'gallery_deprecated'], (layout) ->
+deprecatedLayoutPartnerProfiles = _.map ['gallery_default', 'gallery_deprecated'], (layout) ->
   new Profile fabricate 'partner_profile', owner: fabricate 'partner', profile_layout: layout
 
+institutionLayout = new Profile fabricate 'partner_profile', owner: fabricate 'partner', profile_layout: 'institution'
```

Comment:
#super-minor. This could be called something like `institutionLayoutPartnerProfile` to be clear.